### PR TITLE
[luci-interpreter] Tidy duplicate check

### PR DESCRIPTION
**Pull request by seanshpark**
This will tidy duplicate check in FullyConnected::configure().

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
- found this while working for  #13411

File: compiler/luci-interpreter/src/kernels/FullyConnected.cpp [hunk 1]
```diff
@@ -84,9 +84,6 @@ void FullyConnected::configure()
   const int32_t batch_size = input_shape.num_elements() / weights_shape.dim(1);
   const int32_t num_units = weights_shape.dim(0);
 
-  if (bias())
-    LUCI_INTERPRETER_CHECK(bias()->shape().num_elements() == weights()->shape().dim(0));
```
Comment (seanshpark) [inline compiler/luci-interpreter/src/kernels/FullyConnected.cpp:88]: same as above
```cxx
LUCI_INTERPRETER_CHECK(bias() == nullptr ||
                       bias()->shape().num_elements() == weights_shape.dim(0));
```

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by shs-park — COMMENTED** [review]
👍

**Pull request merged**